xbmc/network/GUIDialogNetworkSetup.cpp

```diff
@@ -226,6 +226,16 @@ void CGUIDialogNetworkSetup::InitializeSettings()
   m_protocols.emplace_back(Protocol{true, true, true, true, false, 22, "sftp", 20260});
 #endif
 
+  for (const auto& addon : CServiceBroker::GetVFSAddonCache().GetAddonInstances())
```

Comment:
should we check for `CServiceBroker::IsBinaryAddonCacheUp()` before access addon cache?

Reply:
done.